lib/msf/core/payload/python/meterpreter_loader.rb

```diff
@@ -35,8 +35,9 @@ def initialize(info = {})
         ),
         OptBool.new(
           'PythonMeterpreterDebug',
-          'Enable debugging for the Python meterpreter'
-        ),
+          'Enable debugging for the Python meterpreter',
+          aliases: ['MeterpreterDebugBuild']
+        )
```

Comment:
I think you'll want to remove this option entirely, and add it as an alias in the base meterpreter options datastore